src/components/CachedImage/CachedImage.tsx

```diff
@@ -42,6 +42,10 @@ class CachedImage extends React.Component<CachedImageProps, CachedImageState> {
   }
 
   componentDidMount() {
+    if("caches" in window) {
+      const { url, url2x } = this.props;
+      window.caches.open('image-cache').then((cache)=> void cache.addAll([ url, url2x ]));
```

Comment:
Do wee need void here? Also could u palce it in a separate method please. And space is missing after `(cache)`